src/users/users.controller.ts

```diff
@@ -44,4 +42,22 @@ export class UsersController {
   ) {
     return this.userService.changeStudentStatus(req.user.id, data);
   }
+
+  @Get('/student-cv/:id')
+  @UseGuards(AuthGuard('jwt'), RoleGuard)
+  @Roles(UserRole.HR)
+  getStudentCv(
+    @Param('id') id: string
+  ): Promise<StudentCvResponse> {
+    return this.userService.getStudentCv(id);
+  }
+
+  @Get('/reserved-students/:recruiterId')
```

Comment:
```suggestion
  @Get('/reserved-students)
```
ten :recruiterId jest chyba niepotrzebny bo i tak go nie używasz tylko bierzesz jego id z requestu